modules/gazelle/0.40.0/MODULE.bazel

```diff
@@ -0,0 +1,72 @@
+module(
+    name = "gazelle",
+    repo_name = "bazel_gazelle",
+    # version is set by the Publish to BCR app.
+)
+
+bazel_dep(name = "bazel_features", version = "0.40.0")
```

Comment:
@kormide I tried to remove `version`, but I somehow still don't make it into this branch: https://github.com/bazel-contrib/publish-to-bcr/blob/bf932c6e44d162715d9d03d21e9b2d2ac853cc55/src/domain/module-file.ts#L42